Compiled dist version does not include classnames module

As per instructions of @nkbt did following:

```
git clone git@github.com:elementalui/elemental.git
cd elemental
npm install
npm run build
```

Took following files from dist folder:
elemental.css
elemental.js

Added these two files to my HTML page header. Executed the page but it complained following:
```
Error: Module name "classnames" has not been loaded yet for context: _. Use require([])
http://requirejs.org/docs/errors.html#notloaded
```
I guess full js output (independent of any dependency) is still not coming.

Reference issue: https://github.com/elementalui/elemental/issues/39#issuecomment-149583456

So I'm trying to play around with elemental UI without using browserify - just use the correct set of precompiled js script tags. I'm not having any luck (admittedly I am new to the whole react and transpiling world), I can't for the life of me get rid of the "Cannot find module 'classnames' " error - I've installed the classname module from bower and inluded the index.js in my script tags, but that doesn't seem to change anything. Is Elemental supposed to be able to be used on it's own, or does it need to be compiled into a bundle using gulp etc before it is usable?

--- a/gulpfile.js
+++ b/gulpfile.js
@@ -8,7 +8,7 @@ var taskConfig = {
 		lib: 'lib',
 		dependencies: [
 			'blacklist',
-			'classnames',
+			'classNames',
 			'history',
 			'react',
 			'react-addons-css-transition-group',

